### DISCUSSION TITLE
Is there a way to build Pyodide such that it support memory64?

### DISCUSSION THREAD

**Discussion by jonah3218** [pyodide/pyodide#4092] [General]
I want to use pyodide to run some machine learning models in the browser with the framework tinygrad. The problem is I am being limited by the small memory available in Pyodide because it is built to tagret 32 bit memory. Is there a way to build pyodide so it uses memory64 instead?

**Comment by ryanking13** [comment]
There is no memory64 build for Pyodide, but probably you can start by adding `-sMEMORY64` in the CFLAGS and LDFLAGS in [Makefile.envs](https://github.com/pyodide/pyodide/blob/main/Makefile.envs).
